cmd/notification.go

```diff
@@ -911,15 +911,12 @@ func (sys *NotificationSys) NetOBDInfo(ctx context.Context) madmin.ServerNetOBDI
 
 		        'x's should be tested, and 'o's should be skipped
 	*/
-	stripPath := func(hostPath string) string {
-		return strings.Split(hostPath, slashSeparator)[0]
-	}
 
 	hostSet := set.NewStringSet()
 	for _, ez := range globalEndpoints {
 		for _, e := range ez.Endpoints {
-			host := stripPath(e.Host)
-			if hostSet.Contains(host) {
+			host := e.Host
```

Comment:
you can use e.Host directly now no need to have a new variable @wlan0 
```go
if !hostSet.Contains(e.Host) {
        sortedGlobalEndpoints = append(sortedGlobalEndpoints, e.Host)
        hostSet.Add(e.Host)
}
```

Reply:
Done